#####################
# Simulation Params #
#####################

initialPosition:
  x: -115
  y: -15
  psi: 0


##################
# Map Parameters #
##################

positionReference:
  latRef: 29.152368
  lonRef: -81.017156
